torchgeo/transforms/transforms.py

```diff
@@ -179,3 +180,77 @@ def forward(
             "input_size": out[0]["input_size"],
             "output_size": out[0]["output_size"],
         }
+
+
+class _ExtractPatches(K.GeometricAugmentationBase2D):
```

Comment:
Only missing a test

Reply:
I don't see any tests for _RandomNCrop btw. Not sure how coverage is passing